test/e2e/Makefile

```
@@ -59,7 +59,7 @@ GINKGO_NOCOLOR ?= false
 
 .PHONY: run
 run: ginkgo ## Run the end-to-end tests
-	cd $(TEST_E2E_DIR); $(GINKGO) -v -trace -tags=e2e -focus="$(GINKGO_FOCUS)" -nodes=$(GINKGO_NODES) --noColor=$(GINKGO_NOCOLOR) . -- \
+	cd $(TEST_E2E_DIR); $(GINKGO) -v -trace -tags=e2e -focus=$(GINKGO_FOCUS) -nodes=$(GINKGO_NODES) --noColor=$(GINKGO_NOCOLOR) $(GINKGO_ARGS) . -- \
```

```suggestion
	cd $(TEST_E2E_DIR); $(GINKGO) -v -trace -tags=e2e -focus="$(GINKGO_FOCUS)" -nodes=$(GINKGO_NODES) --noColor=$(GINKGO_NOCOLOR) $(GINKGO_ARGS) . -- \
```

Removing the quotes from GINKGO_FOCUS breaks that being passed through, causing the regex to break in ginkgo

---

yeah...I should've caught that during the rebase. Thanks!